docs/reference/ois/next/processing.md

```diff
@@ -19,6 +19,22 @@ tags:
 
 # {{$frontmatter.title}}
 
+Processing allows Airnode operators to define custom logic that executes before
+or after an API call. This feature is useful for multiple use cases, including:
+
+- Authentication
+- Data transformation
+- Data aggregation
+- Data validation
+
+## Processing versions
+
+OIS specification has two versions of pre/post processing. Both versions serve
+the same use cases, but the second version is more flexible and convenient.
```

Comment:
By more flexible I mean we can implement the use cases mentioned in https://github.com/api3dao/commons/issues/27

By convenient I mean that processing is just a function now.